### ISSUE TITLE
How to make it work during call?

### ISSUE THREAD

**Issue by anonym24** [veghead/ios-dtmf-decoder#1]
How to make it work during call? I want to call from one iPhone to another and try to detect pressed keyboards

**Comment by Lessica** [comment]
Yes. It's now possible on a jailbroken iOS device or a device with [TrollStore](https://github.com/opa334/TrollStore).

I've made an example here:
https://github.com/Lessica/TrollRecorder/blob/main/cli/dtmf-decoder.mm

Execute `dtmf-decoder` in your terminal and make a phone call from another device.

Press phone keyboard on that device and you will get this:
<img width="566" alt="image" src="https://github.com/user-attachments/assets/23da5ac9-7ae1-41c6-b1d6-a8b430e0d925">
